setup.py

```diff
@@ -44,6 +44,9 @@ def read_req_file(req_type):
         "pytest11": ["dash = dash.testing.plugin"],
     },
     url="https://plotly.com/dash",
+    project_urls={
+        "Source": "https://github.com/plotly/dash",
```

Comment:
```suggestion
        "Documentation": "https://dash.plotly.com",
        "Source": "https://github.com/plotly/dash",
        "Issue Tracker": "https://github.com/plotly/dash/issues",
```
Thanks! Just adding two more links.